crates/e2e/tests/e2e/smart_contract_orders.rs

```diff
@@ -262,5 +305,31 @@ async fn smart_contract_orders(web3: Web3) {
         .call()
         .await
         .expect("Couldn't fetch native token balance");
-    assert_eq!(balance, U256::from(8_972_194_924_949_384_291_u128));
+    assert_eq!(balance, U256::from(7_975_363_884_976_534_272_u128));
```

Comment:
The dramatic change in return amount is because we are getting charged fees twice now (once for each order).